modules/electrophysiology_browser/ajax/get_electrophysiology_session_data.php

```diff
@@ -0,0 +1,327 @@
+/**
+ * Get Electrophysiology Session Data.
+ *
+ * This retrieves the entire electrophysiology session data
+ * for a candidate to be visible on the electrophysiology_session form.
+ *
+ * PHP Version 7
+ *
+ * @category LORIS
+ * @package  Electrophysiology_Session
+ * @author   Muhammad Khan <muhammad.khan@mcin.ca>
+ * @license  LORIS license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+require_once 'ElectrophysioFile.class.inc';
+
+/**
+ * User permission verification:
+ * Only users that have the electrophysiology view all sites permission OR
+ * have the view site permission and belong to candidates session site
+ * are able to fetch the electrophysiology session data.
+ */
+$user      = \NDB_Factory::singleton()->user();
+$timePoint = \NDB_Factory::singleton()->timepoint(
+    intval($_REQUEST['sessionID'])
+);
+
+// if a user does not have the permission to view all sites' electrophsyiology
+// sessions or if a user does not have permission to view other sites' session
+// then display a Forbidden message.
+if (!$user->hasPermission('electrophysiology_browser_view_allsites')
+    && !((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs')))
+    && $user->hasPermission('electrophysiology_browser_view_site'))
+) {
+    header('HTTP/1.1 403 Forbidden');
+    exit;
+}
+
+$response = getSessionData($_REQUEST['sessionID']);
+
+echo json_encode($response);
+
+/**
+ * Get the session data information.
+ *
+ * @param string $sessionID ID of the electrophysiology session
+ *
+ * @return array with the session information
+ */
+function getSessionData(string $sessionID)
+{
+    $db = \NDB_Factory::singleton()->database();
+
+    $query = 'SELECT 
+                DISTINCT(pf.SessionID) 
+              FROM physiological_file pf 
+                LEFT JOIN session s ON (s.ID=pf.SessionID) 
+                LEFT JOIN candidate c USING (CandID)
+                LEFT JOIN psc ON (s.CenterID=psc.CenterID) 
+                LEFT JOIN physiological_output_type pot 
+                  USING (PhysiologicalOutputTypeID) 
+              WHERE 
+                s.Active = "Y" 
+                AND pf.FileType IN ("bdf", "cnt", "edf", "set", "vhdr", "vsm") 
+              ORDER BY pf.SessionID';
+
+    $sessions            = $db->pselect($query, array());
+    $sessions            = array_column($sessions, 'SessionID');
+    $response['patient'] = getSubjectData($sessionID);
+    $response['database']    = array_values(getFilesData($sessionID));
+    $response['sessions']    = $sessions;
+    $currentIndex            = array_search($sessionID, $sessions);
+    $response['nextSession'] = isset($sessions[$currentIndex+1]) ?? '';
+    $response['prevSession'] = isset($sessions[$currentIndex-1]) ?? '';
+
+    return $response;
+}
+
+/**
+ * Get the subject data information.
+ *
+ * @param int $sessionID ID of the electrophysiology session
+ *
+ * @return array with the subject information
+ */
+function getSubjectData($sessionID)
```

Comment:
```suggestion
function getSubjectData($sessionID): array
```

Also add int or string to sessionID, depending on previous comment.

Reply:
done